sentences/sv/light_HassLightSet.yaml

```diff
@@ -3,10 +3,44 @@ intents:
   HassLightSet:
     data:
       - sentences:
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[s | n | ns ] [<ljusintensitet>] till <brightness>"
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <name>[n]  till <brightness> [<ljusintensitet>]"
-          - "<name>[s | n | ns ] [upp | ner ] [till] <brightness> [<ljusintensitet>]"
+          - "[<dimra>] [upp | ner] <name> [<ljusintensitet>] till <brightness>"
+          - "[<dimra>] [upp | ner] <name> till <brightness> [<ljusintensitet>]"
+          - "<name> [upp | ner ] [till] <brightness> [<ljusintensitet>]"
       - sentences:
-          - "[sätt | vrid | dra [på] | skruva | dimmra] [upp | ner ] <alla> <ljuskällor> [<i_på>] <area>[et] [upp | ner ] till <brightness> [<ljusintensitet>]"
+          - "[<dimra>] [upp | ner] [<alla>] <ljuskällor> [<i_på>] <area> [upp | ner ] till <brightness> [<ljusintensitet>]"
+          - "[<dimra>] [upp | ner] [<alla>] <ljuskällor> till <brightness> i <area>"
         slots:
           name: "all"
+
+      - sentences:
+          - "[<dimra>] [upp] <name> [<ljusintensitet>] till <maximal> [<möjliga>]"
```

Comment:
[] around <möjliga> is redundant since <möjliga> contains everything inside it's own [], we should probably keep one set of [] and review the expansion_rules to:
`
  möjliga: "(möjlig[a] | tillgänglig[a])"
`
So it's up to the usage to decide if it should be optional or not.

Reply:
Removed the redundant [] in a new commit. 

@synesthesiam is this method to find the intents for maximum and minimum brightness without having to say 100/1 percent correct usage of the intents?

Reply:
Yes, here's how I would do it in English.
In `_common.yaml` create a new list:

```yaml
lists:
  brightness_min_max:
    values:
      - in: "(max | maximum)"
        out: 100
      - in: "(min | minimum)"
        out: 1
```

Then you reference the list in a sentence:

```yaml
intents:
  HassLightSet:
    data:
      - sentences:
          - "set <name> brightness to [the] {brightness_min_max:brightness}"
```

Note that `{brightness_min_max:brightness}` means to use the `brightness_min_max` list but put the result into the intent slot named `brightness`.

And now this will work:

```sh
python3 -m script.intentfest parse --language en --sentence 'set bedroom lamp brightness to max'
{
  "text": "set bedroom lamp brightness to max",
  "match": true,
  "intent": "HassLightSet",
  "slots": {
    "name": "light.bedroom_lamp",
    "brightness": 100
  }
}
```

Reply:
@synesthesiam That looks like a much cleaner way to do it. I tried to add it to swedish but the tests/linter seems to not like the ":" as i get the error below. But testing with  intentfest as you showed does return the expected responese.

```
Running tests for file light_HassLightSet.yaml

Validation failed

Language: sv
 - sentences/sv/light_HassLightSet.yaml: invalid format: Sentences should only contain words and matching syntax. They should not contain punctuation. @ data['intents']['HassLightSet']['data'][2]['sentences'][0]. Got '[<dimra>] [upp | ner] <name> [<ljusintensitet>] till {brightness_level:brightness} [<möjliga>]'
```